example/app_layout_example/lib/stacking_drawer_example.html

```diff
@@ -0,0 +1,57 @@
+<!--
+ Copyright (c) 2016, the Dart project authors.  Please see the AUTHORS file
+ for details. All rights reserved. Use of this source code is governed by a
+ BSD-style license that can be found in the LICENSE file.
+-->
+<material-drawer class="drawer1" stackable [(visible)]="drawerVisible"
+    [attr.end]="end ? '' : null"
+    [attr.overlay]="overlay ? '' : null">
+  <div *deferredContent>
```

Comment:
You should be able to use the new NG beta attribute `.if` feature (see https://github.com/dart-lang/angular/issues/1058, and https://github.com/dart-lang/angular/blob/master/angular/CHANGELOG.md#new-features-1). E.g.:

```html
<material-drawer class="drawer1" stackable [(visible)]="drawerVisible"
    [attr.end.if]="end"
    [attr.overlay.if]="overlay">
```

Reply:
On lines 7, 8 as well as 15, 16 below.